nodebuilder/tests/reconstruct_test.go

```diff
@@ -81,6 +82,164 @@ func TestFullReconstructFromBridge(t *testing.T) {
 	require.NoError(t, errg.Wait())
 }
 
+/*
+Test-Case: Full Node reconstructs blocks from each other, after unsuccessfully syncing the complete
+block from LN subnetworks. Analog to TestShareAvailable_DisconnectedFullNodes.
+*/
+func TestFullReconstructFromFulls(t *testing.T) {
+	light.DefaultSampleAmount = 10 // s
+	const (
+		blocks = 10
+		btime  = time.Millisecond * 300
+		bsize  = 8  // k
+		lnodes = 12 // c - total number of nodes on two subnetworks
```

Comment:
How much more time higher param take?

Reply:
I couldn't get it to pass with the other parameters, so we decided we should test this scenario in TG and have the swamp be this mild version - since avail tests already show its possible with the other parameter set anyways

Reply:
For clarification, I am convinced the only reason the other swamp tests work with so many light nodes is because the parameters ensure a very high share redundancy. The shares are available from many sources

Reply:
avail tests made me think of diff avail first : )